server/tests/meetup.spec.js

```diff
@@ -23,7 +23,7 @@ describe('User can create a new meetup', () => {
 				tags: 'api, endpoints',
 			})
 			.end((err, res) => {
-				expect(res).to.have.status(200);
+				expect(res).to.have.status(201);
```

Comment:
Noticed that you are only asserting the status code and type of the response. You could do more extensive testing by asserting that the meetup details that were passed in were actually saved to the database. Apply this to other tests